src/Sylius/Bundle/ProductBundle/Resources/config/services.xml

```diff
@@ -65,5 +65,8 @@
             <argument>%sylius.model.product_attribute_value.class%</argument>
             <tag name="doctrine.event_listener" event="postUpdate" lazy="true" />
         </service>
+
+        <service id="sylius.available_product_option_values_resolver" class="Sylius\Component\Product\Resolver\AvailableProductOptionValuesResolver" />
```

Comment:
Just thinking out loud / not really related to this PR: Should we still use service names like "sylius.available_product_option_values_resolver"? Or should we go with "Sylius\Component\Product\Resolver\AvailableProductOptionValuesResolverInterface"? Don't know.

Reply:
It's a matter of taste but I think service id notation like `sylius.available_product_option_values_resolver` is better because you can use this notation to define all the services of your application. Instead having ids like `Sylius\Component\Product\Resolver\AvailableProductOptionValuesResolverInterface` is possibile only when there's only one implementation of the interface and you need only one instance of that implementation in your application. So using FQCN as service ids you'll end up having both notations in your config files and I don't like it.